openapi.yaml

```diff
@@ -18873,11 +18916,354 @@ paths:
+        '200':
+          description: Returns a single placement group object.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PlacementGroup'
       x-code-samples:
         - lang: Shell
           source: >
             curl -H "Authorization: Bearer $TOKEN" \
-              https://api.linode.com/v4/object-storage/transfer/
+                -X GET \
+                https://api.linode.com/v4/placement/groups/528
+        - lang: CLI
+          source: >
+            linode-cli placement view 528
+    put:
+      x-linode-grant: read-write
+      tags:
+      - Placement Groups
+      summary: Placement Group Update
+      description: |
+        Change the `label` for a specific placement group. This is the only value you can update. However, you can [add](#placement-group-add-linode) more compute instances or [remove](#placement-group-remove-linode) existing ones.
+      operationId: updatePlacementGroup
+      x-linode-cli-action: update
+      security:
+      - personalAccessToken: []
+      - oauth:
+        - linodes:read_write
+      requestBody:
+        description: >
+          Include the `label` parameter to update the name of your placement group.
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                label:
+                  $ref: '#/components/schemas/PlacementGroupRequest/properties/label'
+      responses:
+        '200':
+          description: The updated placement group.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PlacementGroup'
+        default:
+          $ref: '#/components/responses/ErrorResponse'
+      x-code-samples:
+      - lang: Shell
+        source: >
+          curl -H "Content-Type: application/json" \
+              -H "Authorization: Bearer $TOKEN" \
+              -X PUT -d '{
+                "label": "PG_Miami_failover_update"
+              }' \
+              https://api.linode.com/v4/placement/groups/528
+      - lang: CLI
+        source: >
+          linode-cli placement update 528 \
+            --label PG_Miami_failover_update
+    delete:
+      x-linode-grant: read_write
+      tags:
+      - Placement Groups
+      summary: Placement Group Delete
+      description: |
+        Deletes a placement group you have permission to `read_write`.
+
+        * Deleting a placement group can't be undone.
+        * All compute instances need to be [removed](#placement-group-unassign) before you can delete a placement group.
+        * If your placement group is non-compliant, you can't delete it. You need to wait for our help to bring it [compliant](/docs/products/compute/compute-instances/guides/placement-groups/#non-compliance-precedence).
+      operationId: deletePlacementGroup
+      x-linode-cli-action:
+        - delete
+        - rm
```

Comment:
Could these actions become `group-delete` and `group-rm` to align with the POST request command?